pkgs/applications/misc/redshift/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchFromGitHub, autoconf, automake, gettext, intltool
 , libtool, pkgconfig, wrapGAppsHook, wrapPython, gobject-introspection
-, gtk3, python, pygobject3, hicolor-icon-theme, pyxdg
+, gtk3, python, pygobject3, hicolor-icon-theme, pyxdg, systemd
```

Comment:
Why does it need `systemd` as a dependency just to install some .ini files?

Reply:
You're right, it doesn't I was just copying from pulseaudio. Removed, and thanks.